src/features/MovieCategories/component.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import './style.scss';
+
+export default class MovieCategories extends Component 
+{
+    // define our state
+    constructor(props)
+{
+        super(props);
+    }
+
+    render() 
+{
+        return (
+	<aside className='movieSection-categories'>
+		<ul className='category-items'>
+			{this.props.categories.map((item, index) => (
+				<h4 
+					key={index} 
```

Comment:
index should be your last resort when you choose what to use as key.
In situation when you work with array and at some point you remove item from this array (from the middle), only index of the last item will change, which will result in the false UI.
you had: [1,2,3,4,5]
you have now : [1,2,4,5]
React will show: [1,2,3,4]

Let's find something better to use instead of that

Reply:
In this scenario the categories array is hardcoded and no .push, .pop, etc. methods are expected to be called on them. I know it's bad practice but made an exception for this simple item